src/BenchmarkDotNet/Toolchains/NativeAot/NativeAotToolchain.cs

```diff
@@ -40,6 +40,14 @@ public class NativeAotToolchain : Toolchain
             .TargetFrameworkMoniker("net9.0")
             .ToToolchain();
 
+        /// <summary>
+        /// compiled as net10.0, targets latest NativeAOT build from the .NET 9 feed: https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet10/nuget/v3/index.json
```

Comment:
minor copy-paste error

```suggestion
        /// compiled as net10.0, targets latest NativeAOT build from the .NET 10 feed: https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet10/nuget/v3/index.json
```